ultraplot/axes/plot.py

```diff
@@ -3459,12 +3459,9 @@ def _apply_scatter(self, xs, ys, ss, cc, *, vert=True, **kwargs):
         ys, kw = inputs._dist_reduce(ys, **kw)
         ss, kw = self._parse_markersize(ss, **kw)  # parse 's'
 
-        # Move _parse_cycle before _parse_color
-        kw = self._parse_cycle(xs.shape[1] if xs.ndim > 1 else 1, **kw)
-
         # Only parse color if explicitly provided
+        infer_rgb = True
         if cc is not None:
-            infer_rgb = True
```

Comment:
This appears to be a change in defaults. Can you add a comment as to why or put it back?

Reply:
I moved this back t o how proplot does it. I initially thought I monkey patched in cycling behavior which broke on the original PR I had with proplot. The colors were cycling but the other properties were not. Not is does all.